src/scores/continuous/standard_impl.py

```diff
@@ -418,3 +419,145 @@ def pbias(
 
     _pbias = 100 * error.mean(dim=reduce_dims) / obs.mean(dim=reduce_dims)
     return _pbias
+
+
+def kge(
+    fcst: xr.DataArray,
+    obs: xr.DataArray,
+    *,
+    reduce_dims: Optional[FlexibleDimensionTypes] = None,
+    preserve_dims: Optional[FlexibleDimensionTypes] = None,
+    scaling_factors: Optional[Union[list[float], np.ndarray]] = None,
+    return_components: bool = False,
+) -> XarrayLike:
+    # pylint: disable=too-many-locals
+    """
+    Calculate the Kling-Gupta Efficiency (KGE) between observed and simulated (or forecast) values.
+
+    KGE is a performance metric that decomposes the error into three components:
+    correlation, variability, and bias.
+    It is computed as:
+
+    .. math::
+        \\text{KGE} = 1 - \\sqrt{\\left[s_\\rho \\cdot (\\rho - 1)\\right]^2 +
+        \\left[s_\\alpha \\cdot (\\alpha - 1)\\right]^2 + \\left[s_\\beta \\cdot (\\beta - 1)\\right]^2}
+
+    .. math::
+        \\alpha = \\frac{\\sigma_x}{\\sigma_y}
+
+    .. math::
+        \\beta = \\frac{\\mu_x}{\\mu_y}
+
+    where:
+        - :math:`\\rho`  = Pearson's correlation coefficient between observed and forecast values as defined in ``scores.continuous.correlation.pearsonr``
```

Comment:
```suggestion
        - :math:`\\rho`  = Pearson's correlation coefficient between observed and forecast values as defined in :py:func:`scores.continuous.correlation.pearsonr`
```
@durgals - at the moment, the API documentation for KGE has a dot point that renders in readthedocs as follows:

![image](https://github.com/user-attachments/assets/d8d4493c-9539-46f0-a7d0-d010831458cb)

That is fine as is. However, if you want to, you can turn `scores.continuous.correlation.pearsonr` into a clickable link, so users can click on it and be taken to the API docstring for `scores.continuous.correlation.pearsonr`.

If you want it to be a clickable link, you can use :py:func:\`scores.continuous.correlation.pearsonr` (as shown above).

If you want to see an example of how this looks and works when rendered, the PBIAS API docstrings in readthedocs includes a clickable link to `scores.continuous.multiplicative_bias` (below is a screenshot).

![image](https://github.com/user-attachments/assets/a4ab1694-7210-4af9-9f7d-8a4e285b313b)

Reply:
@tennlee all good for me. As suggested by @Steph-Chong can you update with the link to Pearsonr?